.github/scripts/generate_binary_build_matrix.py

```diff
@@ -206,6 +206,9 @@ def generate_wheels_matrix(os: str,
         for arch_version in arches:
             gpu_arch_type = arch_type(arch_version)
             gpu_arch_version = "" if arch_version == "cpu" else arch_version
+            # Skip rocm 3.11 binaries for now as the docker image are not correct
+            if python_version == "3.11" and gpu_arch_type == "rocm":
```

Comment:
@albanD Blast from the past: would you happen to remember which component didn't have the "right version" in the docker image? I'm assuming you meant the manylinux image here?

Reply:
IIRC What I meant is that when I modified the builder repo which biulds the docker images, I only updated the linux docker images for cuda.

Reply:
Ah okay, thanks!